CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+- Enabled `nix::poll` for 'freebsd' and 'netbsd' targets
```

Comment:
This should have the same format as other CHANGELOG messages.  Move it into the `Changed` section, and add a link to the PR